app/javascript/packs/validators/charts/skipped_vote_speedometer.js

```diff
@@ -37,7 +34,8 @@ export default {
   },
 
   mounted: function () {
-    let skipped_vote_percent = this.skipped_vote_percent();
+    this.skipped_vote_percent_val =
+      this.skipped_vote_percent(this.validator, this.batch);
```

Comment:
This should be one line

Reply:
Also, you removed the variable that is later used in this script! So it's still broken, just in a new way.
<img width="796" alt="Zrzut ekranu 2023-04-25 o 17 31 02" src="https://user-images.githubusercontent.com/32059143/234327691-e72e1c69-6ecd-4d4b-9329-b0b6f98e18d2.png">

Reply:
Fixed.

Reply:
Still not fixed. I'm taking over now.